site-src/v1alpha2/references/policy-attachment.md

```diff
@@ -0,0 +1,290 @@
+# Policy Attachment
+
+Many implementations of the API support additional functionality that is not
+covered by the API directly. For example, concepts like timeouts, retries, or
```

Comment:
nit: "the API" is a bit abstract in the beginning and qualifying it as "Gateway API" might help set the context.

Reply:
Updated, thanks!